chalicelib/modules/mongo.py

```diff
@@ -205,6 +205,55 @@ def update_document(self, collection, document_id, query, array_filters=None):
             )
             return False
 
+    @add_env_suffix
+    def update_many_documents(self, collection: str, filter_query: dict, update_query: dict, array_filters=None):
```

Comment:
`def update_many_documents(self, collection: str, filter_query: dict, update_query: dict, array_filters=None): -> dict`

You could also potentially typecast the return value too